apps/test/unit/templates/CertificateShareTest.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../util/reconfiguredChai';
+import CertificateShare from '@cdo/apps/templates/CertificateShare';
+
+const defaultProps = {
+  imageUrl: '/certificate-image',
+  printUrl: '/certificate-print',
+  announcement: {
+    image: '/announcement-image',
+    title: 'Title Text',
+    body: 'body text',
+    buttonId: 'button-id',
+    buttonUrl: '/button-url',
+    buttonText: 'button text'
+  }
+};
+
+describe('CertificateShare', () => {
+  let storedWindowDashboard;
+
+  beforeEach(() => {
+    storedWindowDashboard = window.dashboard;
+    window.dashboard = {
+      CODE_ORG_URL: '//code.org'
+    };
+  });
+
+  afterEach(() => {
+    window.dashboard = storedWindowDashboard;
+  });
+
+  it('renders announcement image url relative to pegasus', () => {
```

Comment:
From the work plan, it looks like all certificates will now be hosted by dashboard under studio.code.org, right?  Is pegasus still involved in the flow at all?

Reply:
my understanding based on existing content in announcements.json is that the marketing team expects image urls there to be relative to pegasus, even when the announcement is going to appear on dashboard. I think LP would be best positioned to say whether this should change, but there could be some nuances to work through before changing this pattern.